tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExtensionsMetadataGeneratorTests.csproj

```
@@ -37,7 +38,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="ExistingRuntimeAssemblies.txt">
+    <EmbeddedResource Include="ExistingRuntimeAssemblies-net8.txt">
```

Can use globbing here

```suggestion
    <EmbeddedResource Include="ExistingRuntimeAssemblies-*.txt">
```